apps/daffio/src/app/app.module.ts

```diff
@@ -70,4 +76,16 @@ import { TemplateModule } from './core/template/template.module';
     provideDaffRouterDataServiceConfig(daffioRouterDataServiceConfig),
   ],
 })
-export class AppModule {}
+export class AppModule {
+  constructor(
+    @Inject(PLATFORM_ID) private platformId: object,
+    private injector: Injector,
+  ) {
+    if (isPlatformBrowser(this.platformId)) {
```

Comment:
What are you plans for supporting SSR?

Reply:
hmm if daffio is already built (`npx nx run daffio:build`) you can just run the SSR server through the vscode debug config in `launch.json`.